paddle/go/cmd/master/master.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"flag"
+	"net"
+	"net/http"
+	"net/rpc"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/PaddlePaddle/Paddle/paddle/go/master"
+	"github.com/wangkuiyi/recordio"
+)
+
+const (
+	taskTimeoutDur = 20 * time.Minute
+	taskTimeoutMax = 3
+)
+
+func main() {
+	port := flag.Int("p", 0, "port of the master server")
```

Comment:
@Yancey1989 Good point! Done.
@wangkuiyi I agree that long names are more descriptive in general. In this case, I think it's pretty common that port is specified as `-p`.